tests/unit/spanner_dbapi/test_connection.py

```diff
@@ -239,9 +244,7 @@ def test_run_prior_DDL_statements(self):
                 connection.run_prior_DDL_statements()
 
     def test_context(self):
-        from google.cloud.spanner_dbapi import Connection
-
-        connection = Connection(self.INSTANCE, self.DATABASE)
+        connection = self._make_connection()
```

Comment:
This test has an error: `self.DATABASE` is actually a string, not a `Database` object, so passing it into `Connection` object constructor will cause errors.